content/app/github/commit-page-helper.js

```diff
@@ -0,0 +1,53 @@
+define("content/app/github/commit-page-helper", function(require, exports, module) {
+
+	// imports
+	var FileElementWrapper = require("content/app/github/wrappers/dom-element-wrapper");
+
+	/**
+	 * Helper for accessing and manipulating DOM of commit and pull request page.
+	 */
+	var CommitPageHelper = Class(Object, {
+
+		$statics : {
+
+			FILE_CLASS : 'file',
```

Comment:
Actually I forgot to move this constant into FileElementWrapper. Config is not proper place for this, this should be in class as wrapper class is specially implemented for that element. If element definition changes (and includes className, dom structure, etc.) then wrapper should be changed, not configuration. Config should be used for more general things.